keylime/cloud_verifier_tornado.py

```diff
@@ -472,6 +472,7 @@ def post(self):
                     agent_data["verifier_ip"] = config.get("cloud_verifier", "cloudverifier_ip")
                     agent_data["verifier_port"] = config.get("cloud_verifier", "cloudverifier_port")
                     agent_data["attestation_count"] = 0
+                    agent_data["tpm_clockinfo"] = {"start": True}
```

Comment:
The agent_data needs probably to be JSON for the DB storage, but a nicer way is to either specify a Pickler for `TPMClockInfo` (just calling `to_json()`) or to explicitly construct a `TPMClockInfo` first and then calling `to_json()`. It is currently not clear when there is a dict or a `TPMClockInfo` object.

Reply:
agree, will change